content/en/docs/tasks/observability/logs/otel-provider/index.md

```diff
@@ -0,0 +1,197 @@
+---
+title: OpenTelemetry Provider
+description: This task shows you how to configure Envoy proxies to print access logs with OpenTelemetry Provider.
+weight: 10
+keywords: [telemetry,logs]
+aliases:
+    - /docs/tasks/telemetry/access-log
+    - /docs/tasks/telemetry/logs/access-log/
+owner: istio/wg-policies-and-telemetry-maintainers
+test: yes
+---
+
+After completing this task, you understand how to have your application participate in [Envoy's access logging](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage) with [OpenTelemetry (gRPC) Access Log](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/access_loggers/open_telemetry/v3/logs_service.proto).
+Envoy proxies send access information to [otel-collector](https://github.com/open-telemetry/opentelemetry-collector)'s standard output.
+The standard output of otel-collector's containers can then be printed by the `kubectl logs` command.
+
+{{< boilerplate before-you-begin-egress >}}
+
+{{< boilerplate start-httpbin-service >}}
+
+{{< boilerplate start-otel-collector-service >}}
+
+## Enable Envoy's access logging
+
+Istio offers a few ways to enable access logs. Use of the Telemetry API is recommended
+
+### Using Telemetry API
+
+The Telemetry API can be used to enable or disable access logs:
+
+Run the following script to edit the `MeshConfig`, update the YAML files to add `otel` provider.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n istio-system -f -
```

Comment:
this can be improved if we set `otel` provider by default, but I'm not sure that's the right way. @douglas-reid WDYK?

Reply:
I'm working on a proposal that would eliminate the need to add services to MeshConfig. If that is accepted, it would mean that this step would be replaced by adding a label to the `otel-collector.istio-system` service, which should be a better user experience.

I think we should pursue that (or similar), rather than trying to figure out a one-size-fits-all default provider with a universal service address.

Reply:
actually, I hope we can move all service endpoints out of `MeshConfig`.
In the past, ALS require a static cluster, after https://github.com/envoyproxy/envoy/pull/19193 this's not needed anymore.